README.md

```diff
@@ -1,25 +1,50 @@
-# Coiled Distribution
+# Coiled Runtime
 
```

Comment:
Should we include the CI badges?  Not sure if we can do something like this?
```suggestion

[![Conda Build](https://github.com/coiled/coiled-runtime/actions/workflows/conda.yml/badge.svg)](https://github.com/coiled/coiled-runtime/actions/workflows/conda.yml) [![Linting](https://github.com/coiled/coiled-runtime/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/coiled/coiled-runtime/actions/workflows/pre-commit.yml)
```

Reply:
Adding badges would be good. I held off on this because currently the conda build badge is red. I think this is because the most recent GHA build for `main` was cancelled due to us also pushing a release tag. I'd like to see that fixed before adding badges

Reply:
Whoops, didn't notice that. That makes sense. I'll go ahead and merge this one then